dask/blockwise.py

```diff
@@ -252,6 +252,9 @@ def _out_numblocks(self):
 
         return out_d
 
+    def is_materialized(self):
+        return hasattr(self, "_cached_dict")
+
```

Comment:
Would it make sense to impement `Blockwise.__dask_distributed_pack__`?  Maybe this would be a good task for @rjzamora given his current work on Blockwise

Reply:
Yes, `Blockwise.__dask_distributed_pack__()` is definitely next!

Reply:
But we might need some extra API to handle unpacking of remote data.